### ISSUE TITLE
API for sentrifugo

### ISSUE THREAD

**Issue by jask87** [sapplica/sentrifugo#153]
I want to integrate this with my exsisting LDAP and exsisting timeclock software. Is there an API that i can use to do that? If not what is the best approach to create one.

**Comment by webmarfo18** [comment]
Yes i was wondering if there is a way to integrate this HRIS system with other applications using api.

**Comment by jask87** [comment]
Is there an answer on this? It would be an amazing solution if there was an api.

**Comment by tanist-johnlewis** [comment]
Any news in 2019 on this, the age of the API? :)
